packages/ag-charts-community/src/options/series/topology/mapShapeAccessoryOptions.ts

```diff
@@ -0,0 +1,10 @@
+import type { FillOptions, LineDashOptions, StrokeOptions } from '../cartesian/commonOptions';
+
+export interface AgMapShapeAccessoryThemeableOptions extends FillOptions, StrokeOptions, LineDashOptions {}
+
+export interface AgMapShapeAccessoryOptions extends AgMapShapeAccessoryThemeableOptions {
+    /** Configuration for the Map Accessory. */
+    type: 'map-shape-accessory';
+    /** GeoJSON data. */
+    topology?: any;
```

Comment:
Nit: Can we add a `GeoJSON` alias type for the options contract, to make this more self-describing?